packages/storage-lmdb/source/index.ts

```diff
@@ -11,6 +12,13 @@ export class ServiceProvider extends Providers.ServiceProvider {
 		});
 
 		this.app.bind(Identifiers.Database.RootStorage).toConstantValue(rootStorage);
+
+		const consensusStorage = open({
+			compression: true,
+			name: "consensus",
+			path: join(this.app.dataPath(), "consensus"),
```

Comment:
Use `consensus.mdb` to keep *.mdb files on the  same level. 

Root can also be renamed to `ledger.mdb`.